cmp/dataview/DataView.js

```diff
@@ -22,6 +24,7 @@ export const [DataView, dataView] = hoistCmp.withFactory({
 
     render({model, className, ...props}) {
         const [layoutProps, {rowCls, itemHeight, onRowDoubleClicked}] = splitLayoutProps(props);
+        throwIf(!isNumber(itemHeight), 'Must specify a number for itemHeight in DataView.');
```

Comment:
Minor thing, but should probably use `isFinite` here - just in case its set to NaN as the result of some calc / parsing error